llvm/unittests/IR/DebugInfoTest.cpp

```diff
@@ -1077,6 +1073,11 @@ TEST(MetadataTest, DbgVariableRecordConversionRoutines) {
     !11 = !DILocation(line: 1, column: 1, scope: !6)
 )");
 
+  // For the purpose of this test, set and un-set the command line option
+  // corresponding to UseNewDbgInfoFormat, but only after parsing, to ensure
+  // that the IR starts off in the old format.
+  UseNewDbgInfoFormat = true;
```

Comment:
At the end of this test is `UseNewDbgInfoFormat = false`. Shouldn't it get reset to whatever `UseNewDbgInfoFormat` was before this assignment?